feat: rename to nRF Asset Tracker (#51)
NordicSemiconductor/asset-tracker-cloud-docs: docs/device-cloud-protocol/FourKindsOfData.rst

The four kinds of data
######################

> This sections will provide a general introduction in the way devices communication with the cloud in the nRF Asset Tracker project.

Reviewer: ```suggestion
This section provides a general introduction on the communication of the devices with the cloud in the nRF Asset Tracker project.
```
